### PR TITLE
Deprivilege harobr-log, harbor-db, registry image

### PR DESCRIPTION
**Pull request by reasonerjt**

### CODE REVIEW

File: make/common/mariadb/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ FROM vmware/photon:1.0
 
 RUN tdnf distro-sync -y || echo \
     && tdnf install -y sed shadow procps-ng gawk gzip sudo net-tools \
-    && groupadd -r -g 999 mysql && useradd --no-log-init -r -g 999 -u 999 mysql \
+    && groupadd -r -g 10000 mysql && useradd --no-log-init -r -g 10000 -u 10000 mysql \
```
Comment (yixingjia) [inline make/common/mariadb/Dockerfile]: do we need to run some chown 10000:10000 -R /var/lib/mysql  in the docker-entrypoint.sh file for the db?

Comment (reasonerjt) [reply make/common/mariadb/Dockerfile]: entrypoint.sh of mariadb will call `chown`

https://github.com/vmware/harbor/blob/master/make/common/mariadb/docker-entrypoint.sh#L70

File: make/docker-compose.tpl [hunk 2]
```diff
@@ -6,16 +6,18 @@ services:
...
     ports:
-      - 127.0.0.1:1514:514
+      - 127.0.0.1:1514:10514
     networks:
       - harbor
   registry:
     image: vmware/registry:2.6.2-photon
     container_name: registry
     restart: always
+    read_only: true
```
Comment (yixingjia) [inline make/docker-compose.tpl]: with this restrict will it be difficult for later issue investigation?

Comment (reasonerjt) [reply make/docker-compose.tpl]: Thanks make sense I'll remove this flag from compose template.

However let's make sure the images are ready for `read_only`

**Pull request merged**